### DISCUSSION TITLE
What an awesome tool!

### DISCUSSION THREAD

**Discussion by entusdictus** [jamf/Setup-Manager#7] [Show and tell]
We've piloted Setup Manager in our company as a successor for DEPNotify. 
The transition from DEPNotify was absolutely smooth as the same set of policy triggers could be used in our case. The transition literally took only a few hours and the user experience during device rollout went to a whole new level. 
Localization is a pretty nice feature for us, and it's super-simple to implement. 
Thumbs up for the great job you did with this tool! 

<img width="933" alt="Setup-Manager-CANCOM" src="https://github.com/Jamf-Concepts/Setup-Manager/assets/14052665/8b8878c7-08a9-4e87-ae28-dfcf30f261ae">

**Comment by Jokerface1443** [comment]
WHOOP WHOOP! TOS for Life <3

**Comment by Anandtamil** [comment]
Hi,  can you tell me how to change custom logo

**Reply by scriptingosx** [reply]
https://github.com/jamf/Setup-Manager/blob/main/ConfigurationProfile.md#icon
